templates/default/etcd.initd.erb

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+#
+# etcd      This shell script takes care of starting and stopping
+#             etcd
+#
+# chkconfig: 345 80 30
+# description:  etcd service registry
+
+# processname: etcd
+# pidfile: /var/run/etcd/etcd.pid
+
+# Source function library.
+. /lib/lsb/init-functions
+
+# Source etcd configuration
+if [ -f /etc/default/etcd ] ; then
+ . /etc/default/etcd
+fi
+
+RETVAL=0
+prog="etcd"
+
+start() {
+ # Start daemons.
+ echo -n $"Starting $prog: "
+ PID=`pidof etcd`
```

Comment:
`pidof` isn't going to handle multiple instances of the service running simultaneously. This is probably a non-issue in a production environment, but the `etcd_service` resource is intended to safely configure multiple instances